doc/modules/clustering.rst

```diff
@@ -1582,3 +1582,32 @@ Drawbacks
  *  Caliński, T., & Harabasz, J. (1974). "A dendrite method for cluster
     analysis". Communications in Statistics-theory and Methods 3: 1-27.
     `doi:10.1080/03610926.2011.560741 <http://dx.doi.org/10.1080/03610926.2011.560741>`_.
+
+.. _contingency_matrix:
+
+Contingency Matrix
+----------------------
+
+Contingency matrix reports the intersection cardinality for every
+true/predicted cluster pair. The contingency matrix provides sufficient
+statistics for all clustering metrics where the samples are
+independent and identically distributed and one doesn't need to account
+for some instances not being clustered.
+
+Here is an example::
+
+   >>> from sklearn.metrics.cluster import contingency_matrix
+   >>> x = ["male", "male", "male", "female", "female", "female"]
+   >>> y = ["right_handed", "right_handed", "left_handed", "left_handed",
+   ...      "ambidextrous", "ambidextrous"]
+   >>> contingency_matrix(x, y)
+   array([[2, 1, 0],
+          [0, 1, 2]])
+
```

Comment:
Please explain what the values indicate.

Reply:
I preferred adding them as actual text, instead of in-line comments.